src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -0,0 +1,513 @@
+| iEEGCoordinateSystemDescription     | RECOMMENDED. Freeform text description or link to document describing the iEEG coordinate system system in detail (e.g., “Coordinate system with the origin at anterior commissure (AC), negative y-axis going through the posterior commissure (PC), z-axis going to a mid-hemisperic point which lies superior to the AC-PC line, x-axis going to the right”).                                                                                                                                                                           |
+| iEEGCoordinateProcessingDescription | RECOMMENDED. Has any post-processing (such as projection) been done on the electrode positions (e.g., “surface_projection”, “none”).                                                                                                                                                                                                                                                                                                                                                                                                       |
+| iEEGCoordinateProcessingReference   | RECOMMENDED. A reference to a paper that defines in more detail the method used to localize the electrodes and to post-process the electrode positions. .                                                                                                                                                                                                                                                                                                                                                                                  |
+
+### Recommended 3D coordinate systems
+
+It is preferred that electrodes are localized in a 3D coordinate system (with
+respect to a pre- and/or post-operative anatomical MRI or CT scans or in a
+standard space as specified in BIDS [Appendix VIII](../99-appendices/08-coordinate-systems.md)
+about preferred names of coordinate systems, such as ACPC).
+
+### Allowed 2D coordinate systems
+
+If electrodes are localized in 2D space (only x and y are specified and z is
+n/a), then the positions in this file must correspond to the locations expressed
+in pixels on the photo/drawing/rendering of the electrodes on the brain. In this
+case, coordinates must be (row,column) pairs, with (0,0) corresponding to the
+upper left pixel and (N,0) corresponding to the lower left pixel.
+
+### Multiple coordinate systems
+
+If electrode positions are known in multiple coordinate systems (e.g., MRI, CT
+and MNI), these spaces can be distinguished by the optional `[_space-<label>]`
```

Comment:
(this can be addressed in a new PR) If I have two `_coordsystem.json` or two `_electrodes.tsv` files. One with coordinates in `MNI152Lin` space (as defined in Appendix VIII) and one in a space defined by the subject T1w scan. What values of `_space-` filename keywords these two files should have and how they correspond to allowed values listed in Appendix VII? It might be good to add this example in the specification.

Reply:
I can add an example from the bids-examples motorMiller data (e.g. sub-bp and several others have electrodes in individual subject ACPC/RAS space and in MNI space). If the label in space-<label> is restricted to appendix VII, I would have to remove the space-T1w, since we agreed that T1w is not a well-defined space. I see two options:
1) Just remove space-T1w and resolve this issue later.
2) I could draft a section for iEEG specific coordinate systems for appendix VIII, similar to the MEG section. This would add as few as possible labels to space-<label> that should be used for iEEG electrodes that are not in template coordinate systems.

Reply:
I would for now go with option 1, i.e. remove T1w as allows value for space.
 
Regarding option 2: I just added a description of the individual's ACPC in the appendix , see https://github.com/sappelhoff/bids-specification/pull/9.